resources/lang/en/events/relationship_events/breakup_mates.json

```
@@ -0,0 +1,38 @@
+{
+	"had_fight":[
+		"m_c and r_c had a huge fight and broke up.",
+		"At the same spot in camp they announced their commitment to each other as mates, m_c and r_c announce their break up, looking coldly at each other with anger and hurt in their eyes.",
+		"m_c and r_c have been arguing with each other a lot as of late, making everyone uneasy around them. c_n is secretly relieved when they announce that they're no longer mates.",
+		"m_c and r_c had an excellent relationship and were always at each other's side. But to the Clan's surprise, m_c and r_c had a huge fight in the middle of the clearing, exchanging hurtful words before calling off their relationship for good.",
+		"When a Clanmate asked m_c about r_c, m_c hissed angrily at them, saying {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} want to hear anything about that fox-heart again.",
+		"That's it! m_c's voice rings out through the whole camp with cats watching with unease and curiosity. m_c and r_c had enough of each other's fox-dung after arguing about every little thing for the past moon now. m_c calls off their relationship right there on the spot."
+	],
+	"decided_to_be_friends":[
+		"m_c and r_c realized that they were better off as friends, so they decided to break up.",
+		"After thinking for some time, m_c told r_c that {PRONOUN/m_c/subject} now {VERB/m_c/see/sees} {PRONOUN/r_c/object} as more like a close friend to {PRONOUN/m_c/object} than a mate. It turns out, r_c feels the same way, and they decide to remain close, but not together. As friends, not lovers.",
+		"m_c realizes that {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} have the mentality to keep a romantic relationship right now. After a little talk with r_c, they have decided that they should just be friends, for now at least.",
+		"r_c is an amazing cat, perfect in every way, shape, and form. That's why m_c feels like {PRONOUN/m_c/subject} {VERB/m_c/aren't/isn't} a good enough mate for {PRONOUN/r_c/object} anymore. After a talk with r_c, they've decided to stay friends, giving m_c the incentive to work on {PRONOUN/m_c/self} more.",
```

```suggestion
		"m_c feels like {PRONOUN/m_c/subject} {VERB/m_c/aren't/isn't} a good fit for r_c anymore. After a talk with {PRONOUN/r_c/object}, they've decided to just be friends.",
```
This event came off as a bit more toxic/degrading than really feels appropriate.  I know it's certainly a common trope! But it can easily feel like crossing a line. I've suggested a rewording that helps make it feel more ambiguous and allows the player to interpret in whatever direction they wish.